lib/flat_search/filters.ex

```diff
@@ -0,0 +1,10 @@
+defmodule FlatSearch.Filters do
+  @moduledoc """
+  The filters context
+  """
+
+  alias FlatSearch.Filters.Filter
+
+  def change_filter(%Filter{} = filter), do: Filter.changeset(filter, %{})
+  def change_filter(%Filter{} = filter, _), do: Filter.changeset(filter, %{})
```

Comment:
how about 
`  def change_filter(%Filter{} = filter, params // %{}), do: Filter.changeset(filter, params)` or `\\` not sure 😛 
this way it would not need two def's